src/main/java/mate/academy/springboot/datajpa/repository/ProductRepository.java

```diff
@@ -12,5 +11,5 @@
 public interface ProductRepository extends JpaRepository<Product, Long> {
     List<Product> findProductsByPriceBetween(BigDecimal from, BigDecimal to);
 
-    List<Product> findProductsByCategoryIsIn(Collection<Category> category);
+    List<Product> findProductsByCategoryIdIsIn(Collection<Long> categoriesIds);
```

Comment:
does it work?

Reply:
Yes, I tried it

Reply:
> does it work?

Yes, I tried it

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/113524011/210061258-b7e5039e-6efd-4e1e-bd54-d5bdcdde6104.jpg)